test/f90_correct/len_intrinsic.f90

```diff
@@ -0,0 +1,47 @@
+!** Copyright (c) 2019, Arm Ltd.  All rights reserved.
+
+!** Licensed under the Apache License, Version 2.0 (the "License");
+!** you may not use this file except in compliance with the License.
+!** You may obtain a copy of the License at
+!**
+!**     http://www.apache.org/licenses/LICENSE-2.0
+!**
+!** Unless required by applicable law or agreed to in writing, software
+!** distributed under the License is distributed on an "AS IS" BASIS,
+!** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+!** See the License for the specific language governing permissions and
+!** limitations under the License.
+
+!* Test fix for kind mismatch when intrinsic len is used
+
+! RUN: %flang -S -emit-llvm %s -o - | FileCheck %s -check-prefix=CONVERT
+! RUN: %flang -S -emit-llvm %s -o - | FileCheck %s -check-prefix=NO_CONVERT
+! RUN: %flang -fdefault-integer-8 -S -emit-llvm %s -o - | FileCheck %s -check-prefix=CONVERT_I8
+! RUN: %flang -fdefault-integer-8 -S -emit-llvm %s -o - | FileCheck %s -check-prefix=NO_CONVERT_I8
+subroutine s1(x)
+  character(len=x) :: s1_str
+  integer :: s1_test_len
+  s1_test_len = len(s1_str)
+  print *, s1_test_len
+end subroutine
+! //CONVERT: define void @s1_
```

Comment:
The `//` should be deleted. The correct FileCheck comment prefix should be `! CONVERT:`.

Reply:
done,